Corrfunc/mocks/DDtheta_mocks.py

```diff
@@ -351,6 +351,304 @@ def DDtheta_mocks(autocorr, nthreads, binfile,
+        cells can be up to (thetamax)^3. Only increase if ``rpmax`` is
+        too small relative to the boxsize (and increasing helps the runtime).
+
+    maxbinref: integer (default 3)
+        The maximum ``bin refine factor`` to use along each dimension.
+
+        Runtime of module scales as ``maxbinref^3``, so change the value of
+        ``maxbinref`` with caution.
+
+        Note that ``max_cells_per_dim`` might need to be increased
+        to accommodate really large ``maxbinref``.
+
+    nrepeats: integer (default 3)
+        Number of times to repeat the timing for an individual run. Accounts
+        for the dispersion in runtimes on computers with multiple user
+        processes.
+
+    return_runtimes: boolean (default ``false``)
+        If set, also returns the array of runtimes.
+
+    Returns
+    -------
+    (nRA, nDEC) : tuple of integers
+        The combination of ``bin refine factors`` along each dimension that
+        produces the fastest code.
+
+    runtimes : numpy structured array
+
+        Only returned if ``return_runtimes`` is set, then the return value
+        is a tuple containing ((nRA, nDEC), runtimes). ``runtimes`` is a
+        ``numpy`` structured array containing the fields, [``nRA``, ``nDEC``,
+        ``avg_runtime``, ``sigma_time``]. Here, ``avg_runtime`` is the
+        average time, measured over ``nrepeats`` invocations, spent in
+        the python extension. ``sigma_time`` is the dispersion of the
+        run times across those ``nrepeats`` invocations.
+
+    Example
+    -------
+    >>> from __future__ import print_function
+    >>> import numpy as np
+    >>> import time
+    >>> from math import pi
+    >>> from os.path import dirname, abspath, join as pjoin
+    >>> import Corrfunc
+    >>> from Corrfunc.mocks.DDtheta_mocks \
+        import find_fastest_DDtheta_mocks_bin_refs
+    >>> binfile = pjoin(dirname(abspath(Corrfunc.__file__)),
+    ...                 "../mocks/tests/", "angular_bins")
+    >>> N = 100000
+    >>> nthreads = 4
+    >>> seed = 42
+    >>> np.random.seed(seed)
+    >>> RA1 = np.random.uniform(0.0, 2.0*pi, N)*180.0/pi
+    >>> cos_theta = np.random.uniform(-1.0, 1.0, N)
+    >>> DEC1 = 90.0 - np.arccos(cos_theta)*180.0/pi
+    >>> autocorr = 1
+    >>> best, _ = find_fastest_DDtheta_mocks_bin_refs(autocorr, nthreads,\
+    ...                                               binfile, RA1, DEC1,\
+    ...                                               return_runtimes=True)
+    >>> print(best) # doctest:+SKIP
+    (2, 1)
+
+    .. note:: Since the result might change depending on the computer,
+    doctest is skipped for this function.
+    """
+
+    weights1 = None
+    weights2 = None
+    weight_type = None
+
+    try:
+        from Corrfunc._countpairs_mocks import countpairs_theta_mocks as\
+            DDtheta_mocks_extn
+    except ImportError:
+        msg0 = "Could not import the C extension for the angular "\
```

Comment:
You can call this variable as `msg0`. Similarly for all other occurrences of `msgN` can be renamed to `msg`